tests/Console/E2ETest.php

```diff
@@ -243,6 +243,19 @@ private function installComposerDeps(): void
             $loader->setPsr4($namespace, $paths);
         }
 
+        $mapPsr0 = require 'vendor/composer/autoload_namespaces.php';
+
+        foreach ($mapPsr0 as $namespace => $paths) {
+            foreach ($paths as $path) {
+                if (strpos($path, $vendorDir) !== false) {
+                    // Skip known dependency from autoloading
+                    continue 2;
+                }
+            }
+
+            $loader->set($namespace, $paths);
+        }
+
```

Comment:
Is this for things inside `/usr/share/php/` and so on?

Reply:
not sure I understood the question. The code above is for registering PSR-0 namespaces that are located inside `'vendor/composer/autoload_namespaces.php'`, because without this change the code in `E2ETest.php` loads only namespaces registered from `autoload_psr4.php`

Reply:
Never mind, that's just me before a morning coffee.